app/javascript/components/common/GraphicalIcon.tsx

```diff
@@ -1,9 +1,22 @@
 import React from 'react'
+import { wrap } from '../../utils/wrapper'
 
-export function GraphicalIcon({ icon }: { icon: string }) {
-  return (
-    <svg className="c-graphical-icon" role="presentation">
-      <use xlinkHref={`#${icon}`} />
-    </svg>
-  )
+function GraphicalIconContents({ icon }: { icon: string }) {
+  return <use xlinkHref={`#${icon}`} />
```

Comment:
Maybe:
```suggestion
  return <><use xlinkHref={`#${icon}`} /></>
```

Reply:
@ErikSchierboom As a side note, this is one of the components we said we **don't** need to do this for, because we don't call these icons via the bootloader 🙂

Reply:
Yeah I know, it was just a very easy example for me :D

Reply:
Wrapping it in a fragment is likely not going to fix it. My guess is that react now checks the validity of nesting. So the result still needs to be valid HTML. That means this will likely work for _everything_ *except* web-components and svgs.

Reply:
No, this does indeed not fix it:

```
react-dom.development.js:91 Warning: The tag <use> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.
    in use (at GraphicalIcon.tsx:5)
    in GraphicalIconContents (at application.tsx:128)
```

@iHiD I know the icons are not actually the components where this work should be done for. What are the right components?

Reply:
The `IterationSummary` is the main one atm that's called from both the Bootloader and within React.